apps/api/src/middleware/error-tracking.ts

```
@@ -1,443 +1,351 @@
+import { trace, context as otelContext, SpanStatusCode } from '@opentelemetry/api';
+import { errorTracker, HealthcareErrorContext } from '../services/error-tracking-bridge';
+import { extractHealthcareContext } from '../config/error-tracking';
 
-// Sensitive data patterns for redaction
-const SENSITIVE_DATA_PATTERNS = [
-  // Brazilian documents
-  /\b\d{3}\.\d{3}\.\d{3}-\d{2}\b/g, // CPF
-  /\b\d{2}\.\d{3}\.\d{3}-\d{1}\b/g, // RG
-  /\b\d{3}\.\d{2}\.\d{3}-\d{2}\b/g, // CNPJ
+// Initialize error tracker instance
+const errorTracker = new ErrorTracker();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'errorTracker'. Consider to delete it or rename it.

'errorTracker' is defined here:


(lint/suspicious/noRedeclare)